stream-cipher/src/lib.rs

```diff
@@ -97,10 +97,21 @@ pub trait SyncStreamCipher {
 /// Synchronous stream cipher seeking trait.
 pub trait SyncStreamCipherSeek {
     /// Return current position of a keystream in bytes from the beginning.
-    fn current_pos(&self) -> u64;
+    fn get_current_pos(&self) -> u64;
```

Comment:
Why rename this?

Reply:
I was thinking about adding `try_*` variant for it and `try_current_pos` was a bit strange for my taste. If we'll decide that it's not needed, I probably will revert this renaming.